README.md

```diff
@@ -1,3 +1,3 @@
 # Version
-- Spring v3.2.3
+- Spring v3.2.4
```

Comment:
The reason for updating the minor version would be better if it was in the commit message ;)

Reply:
Ah, there wasn't any significant reason, and I upgraded the version because [Spring Initializr](https://start.spring.io/) no longer supports v3.2.3 😄 
![image](https://github.com/booooomb/api/assets/63195613/99744e07-9787-4c36-ac6d-8f3a5f59f371)

Reply:
But I think Spring Boot is more appropriate, not Spring 😅 Let me fix it~!